### ISSUE TITLE
Active tab has the wrong name

### ISSUE THREAD

**Issue by chrarnold** [gitobsidiantutorial/obsidian-tabs#45]
I am so confused with this, it prevents me from using this addon at all:
When I CTRL + Click multiple notes, the displayed name of the note is always changing, and the active note has a wrong name and changes back to the correct name once a different tab is active. I dont get it, what is the purpose of this, a bug?

**Comment by im-richard** [comment]
To add here, I just opened my own bug report / request ( #46 ) -- from what @chrarnold is describing, it may possibly be what I'm describing on my own submitted issue for **anomaly 2**

**Comment by chrarnold** [comment]
Indeed, looks and sounds similar, thanks for linking! I deactivated the plugin, not usable for me with this behavior unfortunately...I there any way to circumvent this?

**Comment by im-richard** [comment]
> Indeed, looks and sounds similar, thanks for linking! I deactivated the plugin, not usable for me with this behavior unfortunately...I there any way to circumvent this?

Not that I've found quite yet. All the functionality for this is probably in the `.obsidian\plugins\obsidian-tabs\main.js` file which means I'd have to look through the code. I just woke up so my brain isn't exactly functioning yet, but if I do find a quick solution I'll post it. Depends on the amount of time it takes vs having the developer possibly adding this.

**Comment by chrarnold** [comment]
Sorry to ask but: This feature makes the plugin unusable to me, dont others have this or they use the plugin differently?

**Comment by im-richard** [comment]
> Sorry to ask but: This feature makes the plugin unusable to me, dont others have this or they use the plugin differently?

The guy who gave us the tip about doing CTRL + Click ( LaserStefan ) mentioned at the end of the conversation that he wished the plugin had static tabs too. And since we're getting the same issue; I'm going to assume this is either a bug, or how the developer originally coded the plugin to work.

Others can chime in here if they read this, but I'm going to assume at this point that it's something everyone is dealing with.
